src/DSS/DSS.jl

```diff
@@ -142,5 +143,7 @@ for (mv, mv!) in ((:A_ldiv_B,  :A_ldiv_B!),
     end
 end
 
+(\){T}(F::DSSFactor{T}, B::StridedVecOrMat{T}) = A_ldiv_B(F, B)
```

Comment:
Yeah, at the moment I thought `A_ldiv_B` was what actually was getting called in the end, but it is just `\`